cfgov/preprocessed/css/cf-enhancements.less

```diff
@@ -520,6 +520,74 @@ tbody {
 
 
 /* topdoc
+  name: Hide
+  family: cf-core
+  patterns:
+    - name: Utility class
+      markup: |
+        <h1 class="u-hide">
+          Show using JavaScript
+        </h1>
+      codenotes:
+        - .u-hidden;
```

Comment:
Yup! I originally had it as `hidden` because of `u-visually-hidden`, but I don't know that it needs to follow that. Fixed in https://github.com/cfpb/cfgov-refresh/commit/ffdaa50432b29bc1e88c579bcc398562d77ad3ca